packages/mongodb-rag-core/src/makeClassifier.ts

```diff
@@ -173,19 +167,6 @@ export function makeClassifier({
     const classification = Classification.parse(
       JSON.parse(response.functionCall.arguments)
     );
-
-    logger?.appendArtifact(
-      `chatTemplates/classifier-${Date.now()}.json`,
-      stripIndents`
-        <SystemMessage>
-          ${messages[0].content}
-        </SystemMessage>
-        <Classification>
-          ${JSON.stringify(classification)}
-        </Classification>
-      `
-    );
-
     return classification;
```

Comment:
@nlarew i started working on moving the `makeClassifier()` to `mongodb-rag-core`, but the classifier is tightly coupled w/ the RunLogger, that i'm not sure how to do this elegantly.

curious your thoughts how to proceed w/ this

Reply:
Yeah good point this probably should not be coupled and instead by more functional. I can do that separately or you can do it here.

The gist would be removing the logger call from the `makeClassifier()` implementation and instead putting it in the Jira Prompt Response command’s logic after we call the classifier.

Reply:
> The gist would be removing the logger call from the makeClassifier() implementation and instead putting it in the Jira Prompt Response command’s logic after we call the classifier.

that seems reasonable to me. i'm going to close this PR and remove the ticket from the core consolidation epic since it'd require some more refactoring